ansible/roles/openfisca_api_fr/templates/nginx/openfisca-web-api.conf.j2

```diff
@@ -0,0 +1,20 @@
+{# This file will be updated by certbot if SSL is enabled, so that this virtual host listens on port 443. #}
+
+server {
+    listen 80;
```

Comment:
Could we set up [`http2`](http://nginx.org/en/docs/http/ngx_http_v2_module.html)? 🙂

Reply:
Very good idea! I never really did that, I made some quick research and noticed that browsers implement HTTP/2 when TLS is enabled ([source](https://en.wikipedia.org/wiki/HTTP/2#Encryption)). So I implemented it if SSL is enabled in the role.